gitlint/tests/expected/cli/test_cli/test_contrib_1

```diff
@@ -1,3 +1,2 @@
 1: CC1 Body does not contain a 'Signed-off-by' line
-1: CT1 Title does not start with one of fix, feat, chore, docs, style, refactor, perf, test, revert, ci, build: "Test tïtle"
```

Comment:
behavior change: now, if the message is not in the correct format, the validation is ended.